style.css

```diff
@@ -0,0 +1,386 @@
+
+
+.intro-texts{
+	max-width: 65%
+}
+
+.about-me {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-left;
+	flex-wrap: wrap;
+	gap: 50px;
+	align-items: center;
+	max-width: 1000px;
+	width: 100%;
+	margin: 0 auto;
+	background-color: #355C7D;
+	padding: 15px;
+	
+}
+.about {
+	background-color:#355C7D;
+	border: 50px solid #355C7D ;
+	width: 100%;
+}
+
+.about-me-text{
+	max-width:70%;
+
+}
+
+.about-me-text > h2, .portfolio-title-container > h2 {
+	font-size: 3.5rem;
+	letter-spacing: 3px;
+	margin-bottom: 15px;
+}
+
+.about-me-text > p, .portfolio-title-container > h3{
+	font-size: 2rem;
+	width: 100%;
+	letter-spacing: 2px;
+	line-height: 30px;
+}
+
+.full-stack-track-link {
+	font-size: 2rem;
+	text-decoration: none;
+	color: #C06C84;
+}
+
+.full-stack-track-link:visited {
+	color: #C06C84;
+}
+
+.about-me > img {
+	width:20%;
+	border-radius: 3px;
+}
+
+.portfolio {
+	display: flex;
+	justify-content: center;
+	flex-direction: column;
+	align-items: center;
+	width: 100%;
+	margin: 0 auto;
+	border-top: 50px solid  #355C7D ;
+	border-bottom: 50px solid  #355C7D ;
+	background-color: #a1596e;
+
+}
+
+.portfolio-title-container {
+	max-width: 1000px;
+	width: 100%;
+	padding: 15px;
+}
+
+.projects-container {
+	width: 100%;
+	max-width: 1000px;
+	display: flex;
+	flex-direction:column;
+	justify-content: flex-start;
+	flex-wrap: wrap;
+	gap: 40px;
+	padding: 15px;
+}
+
+.project {
+	display: flex;
+	justify-content: space-around;
+	flex-direction: column; 
+	gap: 10px;
+	border: 5px solid #C06C84;
+	border-radius: 2px;
+	background-color: #355C7D;;
+
+	
+}
+.project > h3 {
+	font-size: 2.5rem;
+	margin: 25px;
+	margin-bottom: 0;
+	letter-spacing: 2px;
+}
+
+.project-content{
+	display: flex;
+	flex-direction: row;
+	width: 100%;
+	max-width: 1000px;
+	justify-content: space-between;
+	flex-wrap: no-wrap;
+	
+}
+
+.project-content img {
+	width: 350px;
+	height: 200px;
+	padding: 10px;
+
+}
+
+
+
+.project-words {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	
+}
+
+.project-words > p {
+	font-size: 2rem;
+	padding: 10px;
+	letter-spacing: 3px;
+	line-spacing: 40px;
+}
+
+.project-links-container {
+	display:flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	gap: 20px;
+	margin-right: 30px;
+}
+
+.project-links-container > a {
+	font-size: 1.5rem;
+}
+
+.contact {
+	background-color: #355C7D;
+}
+.contact-outer-container {
+	display: flex;
+	justify-content: flex-start;
+	align-items: flex-start;
+	flex-direction: column;
+	max-width: 1000px ;
+	margin: 0 auto;
+	background-color: #355C7D;
+	padding: 15px;
+}
+
+.contact-outer-container > h2 {
+	font-size: 3rem;
+	margin: 15px;
+	letter-spacing: 3px;
+}
+
+.contact-outer-container > p {
+	font-size: 2.5rem;
+	margin: 15px;
+	letter-spacing: 3px;
+	line-height: 30px;
+}
+
+.contact-container {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	width: 100%;
+}
+
+.contact-container > a> img {
+	max-width: 100px;
```

Comment:
you should change this to width, and also change the `height: 40px` in the media query to be width instead. same result but in a more straightforward, easy to understand mechanism